multi_imbalance/ensemble/soup_bagging.py

```diff
@@ -11,10 +12,6 @@
 from multi_imbalance.utils.array_util import setdiff
 
 
-def fit_clf(args):
```

Comment:
You mentioned that there is a problem with SoupBagging unit tests, and I think this is due to moving this method inside the class.

Reply:
The change didn't help. I have tried use Parallel from joblib instead of multiprocessing (just like that cross_validation in sklearn.model_selection._validation) and it seems that it helps.